src/routes/build/+page.ts

```diff
@@ -0,0 +1,7 @@
+import type { PageLoad } from './$types';
+import { getActiveTeachings } from '$lib/teachings';
+import TEACHINGS from '$lib/teachings';
+
+export const load = (async ({ fetch, _ }) => ({
```

Comment:
```suggestion
export const load: PageLoad = (async ({ fetch, _ }) => ({
```

e togliamo quella di sotto, visto che satisfies non serve qui

Reply:
Sempre preso da altro codice come uso. Farei questo cambio uniformemente in tutte le funzioni `load` allora. La documentazione di SvelteKit fa uso delle annotazioni di tipo
```typescript
/** @type {import('./$types').PageLoad} */
```
ma mi sembra esagerato per noi

Reply:
Si quel tipo di annotazioni è per i file JS. Per TS non sono necessarie. Comunque mentre modifichiamo questa zona è buona idea uniformare senza fare commit da diecimila file